manifests/server/backup.pp

```diff
@@ -109,10 +112,11 @@
   create_resources('class', {
       "mysql::backup::${provider}" => {
         'backupuser'               => $backupuser,
+        'backupcronuser'           => $backupcronuser,
         'backuppassword'           => $backuppassword,
         'backupdir'                => $backupdir,
         'backupdirmode'            => $backupdirmode,
-        'backupdirowner'           => $backupdirowner,
+        'backupdirowner'           => $backupcronuser,
```

Comment:
Any reason that you have changed this line?
Seem's like ti should stay as it is.

Reply:
Yes .. this is directory owner where we keep backups so if this is not set to same user by which we are running cron by default it will be root and you will not able to store backup to that directory location after backups are performed.